controllers/memberstatus/memberstatus_controller_test.go

```diff
@@ -39,6 +38,7 @@ import (
 var requeueResult = reconcile.Result{RequeueAfter: 5 * time.Second}
 
 const defaultMemberOperatorName = "member-operator"
+const defaultMemberOperatorDeploymentName = "member-operator-controller-manager"
```

Comment:
I wonder if we'll find ourselves with 2 deployments when we install the new version of the operator, with this deployment name? (Not a blocker, just a question)

Reply:
I'm really sorry, but I don't really understand what you meant by that question...? I'm somehow slow today morning...

Reply:
> I'm somehow slow today morning...

no worries ;) 
my point is: I believe that once we deploy the operator with these changes, we'll have a new deployment resource called `member-operator-controller-manager` and I wonder what will happen to the current `member-operator` one? are we going to have 2 sibling deployments?

Reply:
Long story short: 
it's even more complicated and the easiest & cleanest way is to uninstall the operators and install them again when we merge the PRs

Reply:
ok ;) I was just wondering how the transition would happen on the clusters. But that's also why we have a staging env 😜

Reply:
The stage doesn't help us in this case much - both clusters use the same channel for updates. But I can do the manual update in stage first and then in prod, indeed.